source/includes/_introduction.md

```diff
@@ -10,8 +10,9 @@ This documentation hopes to cover the range of actions that a user can take, and
 * [Authentication](#authentication) - Guide on how to authenticate with the API and get access to available endpoints.
 * [Prediction](#prediction) - Guide on how to retrieve predictions made from specific processes.
 * [Comparison](#comparison) - Guide on how to retrieve comparisons between predictions and actual events for specific processes.
-* [Subsystem Breakdown](#subsystem-breakdown) - Guide on how to retrieve Subsytem Breakdown data.
-* [Forecast Processes](#forecast-processes) - Guide on how to retrieve Forecast Processes data.
+* [Subsystem Breakdown](#subsystem-breakdown) - Guide on how to retrieve subsystem breakdown data.
+* [Forecast Processes](#forecast-processes) - Guide on how to retrieve data to show what subsystem forecasts are available for each process.
```

Comment:
other way around, it's showing what processes are available for each forecast type